features/polyExplorer/src/screens/entityDetails/entityDetails.css

```diff
@@ -201,41 +209,41 @@
 
 .entity-details-text {
     box-sizing: border-box;
-    width: 100%;
-    max-width: var(--max-width);
+    /* width: 100%; */
+    /* max-width: var(--max-width); */
     margin: 0 auto;
 }
 
 .entity-details-source {
-    transform: translateY(-20px);
+    /* transform: translateY(-20px); */
     color: var(--color-medium);
     flex: 0 0 auto;
+    font-family: "Jost Medium";
     font-size: 12px;
-    margin: 0px auto;
+    margin: 0px 24px;
     width: 100%;
-    padding: 0 30px 12px 30px;
     max-width: var(--max-width);
 }
 
 .explore-data-btn-area {
     margin: 0 auto;
-    padding: 12px 30px 0 30px;
+    padding: 12px 24px 0 24px;
 }
 
 .explore-data-btn {
-    position: fixed;
-    bottom: 0;
+    position: relative;
```

Comment:
I have reviewed this in the iOS Simulator and we used `position: fixed` because in some screens, the "Explore data" button covers the "Info" button. Could you please change it back to 
```
    position: fixed;
    bottom: 0;
    right: 0;
    left: 0;
```
?
(and probably the padding would need to be changed back to)

Reply:
Or maybe there is a better solution that `position: fixed` , but `position: relative` is not solving that issue